{"meaning": "Python provides, \u201ctry\u201d and \u201cexcept\u201d keywords to catch exceptions. The \u201ctry\u201d block code will be executed statement by statement. However, if an exception occurs, the remaining \u201ctry\u201d code will not be executed and the except clause will be executed.", "syntax": "try:\n <body>\n raise Exception()\nexcept Exception as e:\n <exception handling>\nfinally:\n <clean-up>\n", "example": "def divide(x, y):\n    try:\n        # Floor Division : Gives only Fractional Part as Answer\n        result = x // y\n        print(\"Yeah ! Your answer is :\", result)\n    except ZeroDivisionError:\n        print(\"Sorry ! You are dividing by zero \")\n \n\ndivide(3, 2)"}